cmd/tailscaled/tailscaled.go

```diff
@@ -551,6 +552,7 @@ func getLocalBackend(ctx context.Context, logf logger.Logf, logID logid.PublicID
 	if root := lb.TailscaleVarRoot(); root != "" {
 		dnsfallback.SetCachePath(filepath.Join(root, "derpmap.cached.json"), logf)
 	}
+	lb.SetWebLocalClient(&tailscale.LocalClient{Socket: args.socketpath})
```

Comment:
I recall talking about this... this won't work for tsnet, will it?  In my testing with multiple tailscale instances, this also doesn't seem to be connecting to the right one.  I'll try to keep debugging, but something seems not quite right

Reply:
I notice that the peerAPI server and listeners are [set directly](https://github.com/tailscale/tailscale/blob/7cc47c2d981f85c3b461d828c39f145be3256346/ipn/ipnlocal/local.go#L240-L241) on the LocalBackend.  I wonder if there would be value in doing the same for localAPI (or rather, if there is a downside or risk in doing so).  That would allow us to have the listener that we can directly call into from our LocalClient.

Reply:
oh.  well, that might not actually make sense.  There typically isn't a long-lived localapi handler.  It's created, used, and destroyed per-request [as needed](https://github.com/tailscale/tailscale/blob/7cc47c2d981f85c3b461d828c39f145be3256346/ipn/ipnserver/server.go#L201-L207).  But that also tells me that if we have a reference to the localbackend, then we could just create our own localapi.Handler to use for the web client to talk to.

Reply:
I just added a commit to this PR that has webServer maintain its own localapi server, the same way that tsnet does.  See what you think.